applications/application_life_cycle_management/odc-creating-applications-using-developer-perspective.adoc

```diff
@@ -17,7 +17,7 @@ The *Developer* perspective in the web console provides you the following option
 * *Helm Chart*: Explore the *Developer Catalog* to select the required Helm chart to simplify deployment of applications and services.
 
 ifdef::openshift-enterprise,openshift-webscale[]
-Note that certain options, such as *Pipelines*, *Event Source*, and *Import Virtual Machines*, are displayed only when the xref:../../cicd/pipelines/installing-pipelines.adoc#op-installing-pipelines-operator-in-web-console_installing-pipelines[OpenShift Pipelines], xref:../../serverless/admin_guide/installing-openshift-serverless.adoc#serverless-install-web-console_installing-openshift-serverless[OpenShift Serverless], and xref:../../virt/install/installing-virt-web.adoc#virt-subscribing-to-the-catalog_installing-virt-web[OpenShift Virtualization] Operators are installed, respectively.
+Note that certain options, such as *Pipelines*, *Event Source*, and *Import Virtual Machines*, are displayed only when the xref:../../cicd/pipelines/installing-pipelines.adoc#op-installing-pipelines-operator-in-web-console_installing-pipelines[OpenShift Pipelines], xref:../../serverless/admin_guide/install-serverless-operator.adoc#serverless-install-web-console_install-serverless-operator[OpenShift Serverless], and xref:../../virt/install/installing-virt-web.adoc#virt-subscribing-to-the-catalog_installing-virt-web[OpenShift Virtualization] Operators are installed, respectively.
```

Comment:
My instinct would be to include Operator in the text each of the xrefs rather than Operators at the end.  Whether that tickles your fancy, is a "do later", or a "do never" item is up to you.  I also don't know the material, so "do never" might be perfectly OK.

Reply:
+1, I don't actually usually edit or maintain docs outside or Serverless and was just updating the link here, but I agree that would improve readability. I'll update this one, thanks

Reply:
Also noticed the prereqs ID for the next heading was missing, so I've added it